### PR TITLE
Improve podman installation

### PR DESCRIPTION
**Pull request by bruno-fs**
- Improve podman-based installation removing the need to restart quipucords ou postgresql containers
- QoL changes allowing to run scans between VMs (tested the installation only on RHEL 8)
- Bump postgresql version to 14.1 (makefile wasn't updated yet)

### CODE REVIEW

File: qpc_tools/server/ansible/install/roles/server_podman/tasks/start_postgres_container.yml [hunk 1]
```diff
@@ -1,13 +1,9 @@
 ---
 
 - name: Start PostgreSQL container with Podman
-  shell: "podman run --name {{ db_name }} --network=container:{{ server_name }} -e POSTGRES_USER={{ db_user }} -e POSTGRES_PASSWORD='{{ db_password }}' -v qpc-data:/var/lib/postgresql/data -d {{db_image_name}}:{{postgres_version}}"
+  shell: "podman run --name {{ db_name }} -e POSTGRES_USER={{ db_user }} -e POSTGRES_PASSWORD='{{ db_password }}' -v qpc-data:/var/lib/postgresql/data -d {{db_image_name}}:{{postgres_version}}"
```
Comment (gtanzillo) [inline qpc_tools/server/ansible/install/roles/server_podman/tasks/start_postgres_container.yml:4]: Out of curiosity, what was the purpose of `--network=container:{{ server_name }}`?

Comment (bruno-fs) [reply qpc_tools/server/ansible/install/roles/server_podman/tasks/start_postgres_container.yml:4]: allow communication between quipucords server and db without using just `localhost:<container port>`. It conflicts with the `--ports` and is probably the reason why postgres was running after quipucords. Podman already puts all containers on the same network, so I just removed this and explicitly told qpc container what is the IP for postgres container.

File: qpc_tools/server/ansible/install/roles/server_podman/tasks/start_quipucords_container.yml [hunk 2]
```diff
@@ -53,6 +53,5 @@
...
-  become: true
-  ignore_errors: yes
\ No newline at end of file
+  shell: "podman run --name {{ server_name }} -p {{ server_port }}:443 -d -e USE_SUPERVISORD={{ use_supervisord }} -e QPC_SERVER_TIMEOUT={{ server_http_timeout }} -e QPC_DBMS_USER={{ db_user }} -e QPC_DBMS_PASSWORD='{{ db_password }}' -e QPC_DBMS_HOST=$(podman inspect -f '{{ '{{' }} .NetworkSettings.IPAddress {{ '}}' }}' {{ db_name }}) -e ANSIBLE_LOG_LEVEL={{ ansible_log_level }} -e NETWORK_CONNECT_JOB_TIMEOUT={{ connect_job_timeout }} -e NETWORK_INSPECT_JOB_TIMEOUT={{ inspect_job_timeout }} -e QPC_SERVER_USERNAME={{ server_username }} -e QPC_SERVER_USER_EMAIL={{ server_user_email }} -e QPC_SERVER_PASSWORD='{{ server_password }}' -v {{ server_sshkeys_volume }}:/sshkeys{{ dir_mount_selinux }} -v {{ server_data_volume }}:/var/data{{ dir_mount_selinux }} -v {{ server_log_volume }}:/var/log{{ dir_mount_selinux }} -i {{ server_image_and_tag }}"
```
Comment (bruno-fs) [inline qpc_tools/server/ansible/install/roles/server_podman/tasks/start_quipucords_container.yml:56]: @gtanzillo I replaced the `--network` param on postgres container with an env here `-e QPC_DBMS_HOST=$(podman inspect -f '{{ '{{' }} .NetworkSettings.IPAddress {{ '}}' }}' {{ db_name }})`

**Review by gtanzillo — APPROVED** [review]
Nice cleanup, @bruno-fs!

**Pull request merged**